dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+            new File(prioResults).mkdirs();
+        seleResults = dtResults.concat("/selection-results");
+            new File(seleResults).mkdirs();
+        paraResults = dtResults.concat("/parallelization-results");
+            new File(paraResults).mkdirs();
+
+        prioDTLists = dtData.concat("/prioritization-dt-lists");
+            new File(prioDTLists).mkdirs();
+        seleDTLists = dtData.concat("/selection-dt-lists");
+            new File(seleDTLists).mkdirs();
+        paraDTLists = dtData.concat("/parallelization-dt-lists");
+            new File(paraDTLists).mkdirs();
+
+        // Print Output Directories
+        TestPluginPlugin.mojo().getLog().info("Output Directories: " );
+        TestPluginPlugin.mojo().getLog().info("dtResults: " + dtResults);
+        TestPluginPlugin.mojo().getLog().info("dtData: " + dtData);
+        TestPluginPlugin.mojo().getLog().info("prioResults: " + prioResults);
+        TestPluginPlugin.mojo().getLog().info("seleResults: " + seleResults);
+        TestPluginPlugin.mojo().getLog().info("paraResults: " + paraResults);
+        TestPluginPlugin.mojo().getLog().info("prioDTLists: " + prioDTLists);
+        TestPluginPlugin.mojo().getLog().info("seleDTLists: " + seleDTLists);
+        TestPluginPlugin.mojo().getLog().info("paraDTLists: " + paraDTLists);
+
+
+
+        // Extraneous Directories (Not Used, But May Be Useful Later
+        dtTestSource = project.getBuild().getTestSourceDirectory();
+
+        TestPluginPlugin.mojo().getLog().info("Extraneous Directories: " );
+        TestPluginPlugin.mojo().getLog().info("dtTestSource: " + dtTestSource);
+
+        //export DT_ROOT=/home/jgxue2/Test-Second-Bash-Script-V2
+        //export DT_SUBJ_ROOT=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d
+            // Top-most Module
+
+        //export DT_SUBJ_SRC=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core
+            // Desired Target Module
+        //export DT_SUBJ=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target
+            // Desired Module's Target Folder
+        //export DT_LIBS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/dependency/*
+            // Desired Target Module
+        //export DT_CLASS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/classes
+        //export DT_TESTS=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/target/test-classes
+        //export DT_TEST_SRC=$DT_ROOT/elastic-job-lite-old-6d6e460a10b535c149c0f670ec562be027b8808d/elastic-job-lite-core/src/test/java
+    }
+
+    // Compile & Gather The Dependencies Of The Subject
+    private void gatherDependencies(){
+        TestPluginPlugin.mojo().getLog().info("Gathering Dependencies Of The Subject");
+
+        cli.doMain(new String[]{"compile",
```

Comment:
I know it's like this in the scripts, but I don't think there's any reason we need to `compile` and `test-compile` and then `install`, as `install` also compiles.